frontend/src/map/GameMap.js

```diff
@@ -4,6 +4,11 @@ class GameMap {
         this.depth = depth
         this.height = height
 
+        this.tileConfig = {
+            width: 1.0,
+            height: 0.25,
+        }
```

Comment:
Actually, I'm thinking we should keep this separate from the map itself. It would be nice if we could keep all integer coordinates in the map realm and then the TileConfig is used by the renderer to place things appropriately.